MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -367,10 +395,14 @@ void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel( vtkMRMLMarkups
     {
       // draw a "square" around the line (make it a rectangular prism)
       vtkSmartPointer<vtkRegularPolygonSource> squareSource = vtkSmartPointer<vtkRegularPolygonSource>::New();
-      double extrusionMagnitude = ComputeSurfaceExtrusionAmount( cleanedPoints ); // need to give some depth
       squareSource->SetCenter( 0.0, 0.0, 0.0 );
+      double extrusionMagnitude = ComputeSurfaceExtrusionAmount( smallestBoundingExtentRanges ); // need to give some depth
       squareSource->SetRadius( extrusionMagnitude );
       squareSource->SetNumberOfSides( 4 );
+      double lineAxis[ 3 ] = { 0.0, 0.0, 0.0 }; // temporary values
+      const int LINE_AXIS_INDEX = 0; // The largest (and only meaningful) axis is in the 0th column
+      // the bounding axes are stored in the columns of transformFromBoundingAxes
+      GetNthAxisInMatrix( transformFromBoundingAxes, LINE_AXIS_INDEX, lineAxis ); 
```

Comment:
I'll rename to NthColumn. I would prefer to keep the function. I made it in order to compartmentalize how the axes are accessed, and also so the reader doesn't necessarily have to burden him/herself with the lower-level detail.